content/en/index.md

```diff
@@ -16,7 +16,7 @@ It’s free to use for all developers and software providers, with Sigstore’s
 
 * Sign software artifacts with [Cosign](/cosign/overview/) — [Quick start](/cosign/overview/#quick-start)
 * Sign Git commits with [Gitsign](/gitsign/overview/) - [Quick start](/gitsign/overview/#quick-start)
-* Verify entries with [Rekor](/rekor/CLI/#verify-proof-of-entry)
+* Verify entries with [Cosign](/cosign/verify)
 * Use the [policy controller](/policy-controller/overview/) to enforce Kubernetes policies
 * Use the [Fulcio Certificate Authority](/fulcio/overview/)
```

Comment:
I'd drop this too, doesn't seem necessary for the initial landing page

Reply:
ok, I will take it completely out.